mbotmake2/transformers/thumbnails.py

```diff
@@ -19,14 +19,14 @@ class PNGImage:
 
 class ThumbnailDecoder(NodeVisitor):
     def visit_Document(self, _, visited_children) -> list[PNGImage]:
-        _, thumbnails = visited_children
+        thumbnails = visited_children
```

Comment:
The PEG grammar is capable of capturing the first few lines that are not Thumbnails. So, instead of writing our own parsing logic to seek the first occurrence of `begin thumbnail`, we can program the PEG grammar to seek the thumbnails for us.